lib/katello/tasks/upgrades/4.0/sync_noarch_content.rake

```diff
@@ -0,0 +1,13 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.0' do
+      desc "Synchronize noarch content in candlepin."
+      task :sync_noarch_content => ['environment'] do
```

Comment:
This migration looks good! Can you add a test? You should be able to find some examples of tests for rake tasks existing already